lib/utils/webpack.config.js

```diff
@@ -368,12 +368,12 @@ module.exports = (program, directory, suppliedStage, webpackPort = 1500, routes
         config.loader('less', {
           test: /\.less/,
           exclude: /\.module\.less$/,
-          loader: ExtractTextPlugin.extract(['css?minimize', 'less']),
+          loader: ExtractTextPlugin.extract(['css?minimize', 'postcss']),
         })
         config.loader('sass', {
           test: /\.(sass|scss)/,
```

Comment:
Oh, understood. I figured out the build first. Odd that it worked fine when it didn't with in develop. I'll make that change though.